hkmc2/shared/src/main/scala/hkmc2/codegen/HandlerLowering.scala

```diff
@@ -152,12 +152,14 @@ class HandlerLowering(using TL, Raise, Elaborator.State, Elaborator.Ctx):
     // for readability :)
     case class PartRet(head: Block, states: Ls[BlockState])
 
-    // returns (truncated input block, child block states)
-    // TODO: don't split within Match, Begin and Labels when not needed, ideally keep it intact. Need careful analysis for this
-    // blk: The block to transform
-    // labelIds: maps label IDs to the state at the start of the label and the state after the label
-    // jumpTo: what state End should jump to, if at all 
-    // freshState: uid generator
+    /* 
+    * returns (truncated input block, child block states)
+    * TODO: don't split within Match, Begin and Labels when not needed, ideally keep it intact. Need careful analysis for this
```

Comment:
Part of my comment was meant to distinguish between the TODO and the parameter enumeration. This is why I added ticks in front of each parameter and a period at the end of the TODO. Your change misses out on both things.